superset/assets/src/dashboard/stylesheets/filter-indicator.less

```diff
@@ -0,0 +1,97 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+.dashboard-filter-indicators-container {
+  position: absolute;
+  z-index: 10;
```

Comment:
is a z index absolutely needed here? Have we ensured that 10 is appropriate compared to all other possible z indexes?

Reply:
be honest, i don't know how to make ensured 10 is appropriate. Just by manual test.

Reply:
got it, we should probably make an issue to manage the z-indexes better at some point. this is fine for now though